src/byzerllm/saas/sparkdesk/__init__.py

```diff
@@ -60,28 +61,29 @@ def create_url(self):
         # 此处打印出建立连接时候的url,参考本demo的时候可取消上方打印的注释，比对相同参数时生成的url与自己代码生成的url是否一致
         return url
 
-class SparkDeskAPI:
+class CustomSaasAPI:
+
+    def __init__(self, infer_params: Dict[str, str]) -> None:
+        self.appid: str = infer_params.get("saas.appid","")
+        self.api_key: str = infer_params.get("saas.api_key","")
+        self.api_secret: str = infer_params.get("saas.api_secret","")
+        self.gpt_url: str = infer_params.get("saas.gpt_url","")
```

Comment:
When the user didn't configure git_url/domain , what will happen?

Reply:
I will give domain a default value